introduction/payment-driver-getting-started.md

```diff
@@ -0,0 +1,81 @@
+# Getting started to write L2 payment driver for Yagna
```

Comment:
It does not have to be a L2 driver, they can also implement a paypal driver if they want ;)